src/test/java/com/adyen/PaymentMethodsRequestTest.java

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
This class not needed since we will not use ApplicationInfo in PaymentMethodRequest class.

Reply:
Reverted.